distributions/standalone/assembly-whitelist.txt

```diff
@@ -206,11 +206,11 @@
 /mule-standalone-${productVersion}/lib/opt/async-http-client-1.9.31.jar
 /mule-standalone-${productVersion}/lib/opt/axiom-api-1.2.5.jar
 /mule-standalone-${productVersion}/lib/opt/axiom-impl-1.2.5.jar
-/mule-standalone-${productVersion}/lib/opt/bcmail-jdk16-1.46.jar
-/mule-standalone-${productVersion}/lib/opt/bcpg-jdk16-1.46.jar
-/mule-standalone-${productVersion}/lib/opt/bcprov-ext-jdk16-1.46.jar
-/mule-standalone-${productVersion}/lib/opt/bcprov-jdk16-1.46.jar
-/mule-standalone-${productVersion}/lib/opt/bctsp-jdk16-1.46.jar
+/mule-standalone-${productVersion}/lib/opt/bcmail-jdk15on-1.46.jar
```

Comment:
the 1.46 is the last version of the *-jdk16 artifacts. 

In the description of the *-jdk15on artifacts says: "This jar contains JCE provider and lightweight API for the Bouncy Castle Cryptography APIs for JDK 1.5 to JDK 1.8."